RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,79 @@
+# Bitcoin Definitions
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 26 Feb. 2019
+
+# Table of contents
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+- [Registry extension](#registry-extension)
+
+## Description
+
+Introduces definitions to allow the execution of RFC-003 for Bitcoin asset on the [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+TODO: Decide we if do 1 RFC that introduce general Bitcoin terms + RFC003 or 2 RFCs (one for each)
+
+This includes the exact Bitcoin scripts to be used.
+
+TODO: Do we want p2sh(p2wsh) support? (more wallets support it)
+
+TODO: Do we want p2sh native support? (more wallets support it + more straightforward than p2sh(p2wsh))
```

Comment:
follow-up of https://github.com/comit-network/RFCs/pull/34#discussion_r260103652

@bonomat What I am saying is that not all Bitcoin Core wallet support P2WSH so it is worth considering P2SH.

Reply:
True, but I would not implement it right now and would only update this document once we support it because we implemented it for BCash.

Reply:
That assumes that we would update for Bitcoin Cash before we finalize this document.
If that's the timeline you see then fair enough.

Reply:
With _finalize_ you mean we cannot come back and revise this document? Why can't we write that we only support P2WSH and in 6 months when we do have BCash support we update this RFC?

Reply:
Can't update the RFC once we mark the RFC as final

Reply:
See above.